dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntityRequestParamsMapper.java

```diff
@@ -153,6 +155,60 @@ public TrackedEntityOperationParams map(
     return builder.build();
   }
 
+  private void validateRequestParams(TrackedEntityRequestParams params) throws BadRequestException {
+
+    String violation = null;
```

Comment:
Here we are declaring a variable and assign every time a different error.
Can we just remove the variable and throw the exception in the if statements?